### PR TITLE
Updates to Kubernetes Metadata docs

### PR DESCRIPTION
**Pull request by AaronCrawfis**
1. Split Kubernetes installation into second page
2. Simplify snippets and example
3. Switch to Deployment description
4. Make example have more self-describing keys/values

### CODE REVIEW

File: docs/content/operations/platforms/kubernetes/kubernetes-install.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (AaronCrawfis) [inline docs/content/operations/platforms/kubernetes/kubernetes-install.md:1]: This content wasn't really part of "supported clusters", so I moved it out into a dedicated page on Kubernetes installation

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md [hunk 2]
```diff
@@ -3,93 +3,110 @@ type: docs
...
 
-You can set labels and annotations on an environment, application, or container using the Kubernetes metadata extension. The Kubernetes objects output from your resources (Deployments, Pods, etc.) will get the defined metadata.
+Project Radius enables you to retain or use your own defined tagging scheme for Kubernetes resources using Kubernetes labels and annotations. This enables users to incrementally adopt Radius for microservices built in the Kubernetes ecosystem using Kubernetes metadata concepts.
 
-### Example
-Here is an example of how to set labels and annotations at the environment layer. All resources within the environment with Kubernetes object outputs will gain this metadata:
+You can set labels and annotations on an environment, application, or container using the Kubernetes metadata extension. The Kubernetes objects output from your resources (_Deployments, Services, etc._) will now have the defined metadata.
 
-{{< rad file="snippets/env.bicep" embed=true >}}
+{{< tabs Environment Application Container >}}
```
Comment (AaronCrawfis) [inline docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md:21]: Made these an inline tab group so users can see how the extension looks on each resource type, before diving into the specific behavior.

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md [hunk 3]
```diff
@@ -3,93 +3,110 @@ type: docs
...
-Container -> Dapr sidecar extension -> Manual scale extension -> Kubernetes metadata extension
+1. Dapr sidecar extension 
+1. Manual scale extension
+1. Kubernetes metadata extension
+
+### Conflicts
+
+When labels/annotation have the same set of key(s) added by two or more extensions, the final value is determined by the last extension that is processed.
+
+## Example
```
Comment (AaronCrawfis) [inline docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md:71]: I combined the example into one flow and one file, and made the keys/values a little more self-describing so users don't need to track what values can from which layer

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md (hunk 2 above)
Comment (Reshrahim) [reply docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md:21]: This is good and the user has examples on how this works against different resources

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md [hunk 4]
```diff
@@ -3,93 +3,110 @@ type: docs
...
 
-#### Example
-Lets consider an example where the service team wants to override the contact information that was set at the environment level.
-The infrastructure operator sets up a generic contact at the environment resource for troubleshooting
+1. Environment
+1. Application
+1. Container
 
-{{< rad file="snippets/override.bicep" embed=true marker="//ENV" >}}
+### Conflicts
```
Comment (Reshrahim) [inline docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md]: I wouldn't want to call this a conflict instead it is an overriding behavior that MBCP wants. Also feel that the example of infra ops setting up certain fields and dev overriding them helped portray the use case better

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md (hunk 3 above)
Comment (Reshrahim) [reply docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md:71]: This helps remove redundancy but feel that the example I had before portrays a real-time use case that can resonate well with the audience

**Review by Reshrahim — CHANGES_REQUESTED** [review]

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md (hunk 3 above)
Comment (AaronCrawfis) [reply docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md:71]: Yeah I was going back and forth myself on this. The tradeoff is definitely between real-world example to show _why_ labels and annotations can be used, vs. teaching _how_ labels and annotations can be used. Perhaps we can find a way to layer in both, to both teach the why and the how in one example. (why I kept the prometheus annotation in the example) With the prior example wasn't sure why a team would want to label the team name and override that. Perhaps we can still leverage the prometheus example to have the developer override a default port or default value

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md (hunk 4 above)
Comment (AaronCrawfis) [reply docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md]: Ah gotcha. Yeah let's definitely come up with a way to include a real world example. For the heading totally get the term preference, although conflict resolution is also a common term for features like this. Perhaps:

```suggestion
### Conflicts and overrides
```

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md [hunk 5]
```diff
@@ -3,93 +3,110 @@ type: docs
...
 
-The developer or any engineer can choose to override this contact information at the container level as they might be supporting only a certain service.
+In the case where layers have conflicting keys (_i.e. Application and Contaienr both specify a `myapp.team.name` label_), the last level to process wins out (container). The metadata specified on the container will override the metadata specified on the application or environment.
```
Comment (AaronCrawfis) [inline docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md]:
```suggestion
In the case where layers have conflicting keys (_i.e. Application and Contaienr both specify a `myapp.team.name` label_), the last level to process wins out and overrides other values (container). The metadata specified on the container will override the metadata specified on the application or environment.
```

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md (hunk 3 above)
Comment (Reshrahim) [reply docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md:71]: So a customer told us that their administrator sets up some labels and annotations and the service level engineer overrides them and they also mentioned that this is for troubleshooting/ auditing. So what I did is to translate this use case into an example where the infra ops sets up a support contact/ team name at a generic level for all the apps and services in the environment and then developer or the team owning say the frontend service can override the generic contact info with his team contact at the container level

Comment (AaronCrawfis) [reply docs/content/operations/platforms/kubernetes/kubernetes-metadata/index.md:71]: Added back in the real-world example with some simplifications, as well as an explanation

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/snippets/env.bicep [hunk 6]
```diff
@@ -14,34 +13,59 @@ resource env 'Applications.Core/environments@2022-03-15-privatepreview' = {
...
-}
+    environment: env.id
+    extensions: [
+      {
+        kind: 'kubernetesMetadata'
+        labels: {
+          'key1': 'appValue1'
+          'key2': 'appValue2'
+          'team.contact.name': 'Operations'
+          'team.contact.email': 'ops@example.com'
```
Comment (Reshrahim) [inline docs/content/operations/platforms/kubernetes/kubernetes-metadata/snippets/env.bicep]:
```suggestion
          'team.contact.email': 'ops'
```
'@' is not supported by Kubernetes labels regex

File: docs/content/operations/platforms/kubernetes/kubernetes-metadata/snippets/env.bicep [hunk 7]
```diff
@@ -14,34 +13,59 @@ resource env 'Applications.Core/environments@2022-03-15-privatepreview' = {
...
-      image: 'radius.azurecr.io/tutorial/webapp:edge'
+      image: 'myregistry/mycontainer:latest'
     }
+    extensions: [
+      {
+        kind: 'kubernetesMetadata'
+        labels: {
+          'key2': 'containerValue2'
+          'team.contact.name': 'Frontend'
+          'team.contact.email': 'frontend-eng@example.com'
```
Comment (Reshrahim) [inline docs/content/operations/platforms/kubernetes/kubernetes-metadata/snippets/env.bicep]:
```suggestion
          'team.contact.email': 'frontend-eng'
```

**Pull request merged**